draft-ietf-spice-sd-cwt.md

```diff
@@ -355,8 +355,7 @@ claim (true), the `region` claim (California), and the other element in the
 ]>>
 ~~~
 
-The Holder will also typically fetch a nonce from the Verifier to prevent
-replay.
+The Holder MAY also fetch a nonce from the Verifier to prevent replay.
```

Comment:
```suggestion
The Holder MAY fetch a nonce from the Verifier to prevent replay, or obtain a nonce acceptable to the verifier through a process similar to the one described in draft-ietf-httpbis-unprompted-auth-12.
```

Reply:
We might also consider a reference to the nonce endpoint as described here: https://github.com/openid/OpenID4VCI/pull/381/files

Reply:
I wonder, since this is related to VC, if it shouldn't go in a "profile" document (for VC in this case). Regarding nonces we could however add a section in the "Security Considerations" and incorporate sane recommendations (size, expiry, reuse etc..) by inspiring us from this draft

Reply:
I consider nonce management part of safely executing cnf, which is a layer above profiles that need confirmation... But you are right that many app protocol flows will probably need to support nonces in order to support the protocol.